### ISSUE TITLE
Binding Threads To Specific PCI Slot Or Card

### ISSUE THREAD

**Issue by tderoock** [fireice-uk/xmr-stak-amd#114]
Hi,

I hope this is the correct place to raise a request.
I am currently running a rig with 5x RX580 and 1 RX Vega 56.
This is working alright, but i should be able to get more out of the vega card.
The way i see people doing this, is by running only the vega card and assign 2 threads to the pool config. 

cfr:

"gpu_threads_conf" : [
    { "index" : 0, "intensity" : 1600, "worksize" : 8, "affine_to_cpu" : false },
    { "index" : 1, "intensity" : 1200, "worksize" : 8, "affine_to_cpu" : false },
],

Now, when only running one card, it is obvious that those two threads are assigned to that card.
But when running multiple cards, of different types, it gets more complex.
It appears that i am getting memory allocation errors, because those threads with elevated intensity are being assigned to cards which cannot handle that intensity.

So basically, is there a way of assiging threads / intensity to a specific card?

**Comment by rumkuhgel** [comment]
Yes, using the index: 0 is the first card in your rig, 1 the second etc.
So if you want to use two threads per card, you have to put the index on the same value, e.g. 0 for one of your cards.
